desktop/src/backends/fscommand.rs

```diff
@@ -0,0 +1,36 @@
+use crate::custom_event::RuffleEvent;
+
+use ruffle_core::external::FsCommandProvider;
+use std::cell::Cell;
+use std::rc::Rc;
+use winit::event_loop::EventLoopProxy;
+use winit::window::{Fullscreen, Window};
+
+pub struct DesktopFSCommandProvider {
+    pub event_loop: EventLoopProxy<RuffleEvent>,
+    pub window: Rc<Window>,
+    pub is_fullscreen: Cell<bool>,
+}
+
+impl FsCommandProvider for DesktopFSCommandProvider {
+    fn on_fs_command(&self, command: &str, _args: &str) -> bool {
+        let _ = match command {
+            "quit" => {
+                let _ = self.event_loop.send_event(RuffleEvent::ExitRequested);
```

Comment:
Hmm I'm a bit unsure about this. There's cases where I can see that closing the window might be preferred - I guess if you opened it specifically for one swf. But every other time I think I'd expect that the movie just stops playing, and I can reopen something else. It's weird for a movie to have control over the host application like that.

Reply:
I did this to match FP's behavior (it closes the window)- this is commonly used in conjunction with `fullscreen`, to make it easier for the user to exit out of a fullscreen window (probably more useful in Ruffle, since you can't press `Esc` to get out of fullscreen, you have to use the less-known `Alt`+`Enter`)